core/src/main/java/org/apache/struts2/components/Set.java

```diff
@@ -83,6 +83,8 @@
 public class Set extends ContextBean {
     protected String scope;
     protected String value;
+    protected boolean doNotTrimBody;
```

Comment:
What about using `trimBody` with default set to `true`?

Reply:
The requested change (switch to positive value with a default of true) has been made with the latest commit.